src/util/apiHelpers.ts

```diff
@@ -76,7 +76,36 @@ const uri = (() => {
 const possibleTypes = {
   TaxonomyEntity: ["Resource", "Topic"],
   SearchResult: ["ArticleSearchResult", "LearningpathSearchResult", "NodeSearchResult"],
-  FolderResourceMeta: ["ArticleFolderResourceMeta", "LearningpathFolderResourceMeta"],
+  FolderResourceMeta: [
+    "ArticleFolderResourceMeta",
+    "AudioFolderResourceMeta",
+    "ConceptFolderResourceMeta",
+    "ImageFolderResourceMeta",
+    "LearningpathFolderResourceMeta",
+    "TopicFolderResourceMeta",
+    "VideoFolderResourceMeta",
+  ],
+};
+
+const toResourceMeta = (type: string) => {
```

Comment:
Stemmer egentlig for at vi sletter hele `Query`-delen av `typePolicies`. Jeg har ikke sett noen store ulemper ved å gjøre det.